composer.json

```diff
@@ -27,5 +36,8 @@
             "AnalyticsReporting",
             "Oauth2"
         ]
+    },
+    "require-dev": {
+        "matomo-org/matomo-coding-standards": "dev-master"
```

Comment:
I'm not sure if requiring the coding standard directly in the repo is useful. We have a lot of plugins that don't even use composer, so for them that would be a total overhead.
We could also simply run a `composer require` for the package within the github action to ensure its available there. (Note: We might need to publish the coding style on packagist to be able to require it on command line)

Reply:
I will try without adding in composer and see how does it look

Reply:
Done and it works without publishing the repo